app/controllers/emails_controller.rb

```diff
@@ -86,16 +87,20 @@ def unsubscribe
 
   def unsubscribed
     @email = Email.find_by email: email_params[:email]
-    @email[:status] = 0
-    @email[:unsubscribed] =  DateTime.current
+    Rails.logger.debug @email
```

Comment:
TODO: Remove logging statements and come up with a better logging solution so that I can be notified of errors on prod when they occur.